tests/test_cli.py

```diff
@@ -221,65 +230,72 @@ def test_gml_bbox(script_runner):
     result = ret.stdout
     bboxList = parse_coordinates(result)
     assert bboxList == pytest.approx([-17.542069, 32.39669, -6.959389, 39.301139])
+    assert "4326" in result
 
 
 def test_gml_time(script_runner):
     ret = script_runner.run('geoextent', '-t', 'tests/testdata/gml/clc_1000_PT.gml')
     assert ret.success, "process should return success"
     assert ret.stderr == '', "stderr should be empty"
-    assert "['2005-12-31', '2013-11-30']" in ret.stdout,  "time value is printed to console"
+    assert "['2005-12-31', '2013-11-30']" in ret.stdout, "time value is printed to console"
 
 
 def test_gml_only_one_time_feature_valid(script_runner):
-    ret = script_runner.run('geoextent','-t', 'tests/testdata/gml/mypolygon_px6_error_time_one_feature.gml')
+    ret = script_runner.run('geoextent', '-t', 'tests/testdata/gml/mypolygon_px6_error_time_one_feature.gml')
     assert ret.stdout
     assert "'tbox': ['2012-04-15', '2012-04-15']" in ret.stdout, "time value is printed to console"
 
 
 def test_shp_bbox(script_runner):
     ret = script_runner.run('geoextent', '-b', 'tests/testdata/shapefile/Abgrabungen_Kreis_Kleve_Shape.shp')
     assert ret.success, "process should return success"
-    assert "'bbox': None" in ret.stdout
+    assert "'bbox'" not in ret.stdout
 
 
 @pytest.mark.skip(reason="multiple input files not implemented yet")
 def test_multiple_files(script_runner):
     ret = script_runner.run('python', 'geoextent',
-        '-b', 'tests/testdata/shapefile/Abgrabungen_Kreis_Kleve_Shape.shp', 'tests/testdata/geojson/ausgleichsflaechen_moers.geojson')
+                            '-b', 'tests/testdata/shapefile/Abgrabungen_Kreis_Kleve_Shape.shp',
+                            'tests/testdata/geojson/ausgleichsflaechen_moers.geojson')
     assert ret.success, "process should return success"
     assert ret.stderr == '', "stderr should be empty"
-    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bboxes and time values of all files inside folder, are printed to console"
-    assert "[6.574722, 51.434444, 4.3175, 53.217222]" in ret.stdout, "bboxes and time values of all files inside folder, are printed to console"
-    assert "[292063.81225905, 5618144.09259115, 302531.3161606, 5631223.82854667]" in ret.stdout, "bboxes and time values of all files inside folder, are printed to console"
+    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout,\
+        "bboxes and time values of all files inside folder, are printed to console"
+    assert "[6.574722, 51.434444, 4.3175, 53.217222]" in ret.stdout, \
+        "bboxes and time values of all files inside folder, are printed to console"
```

Comment:
The assert message does not match what is asserted - no time values checked here!